src/EnergyPlus/CMakeLists.txt

```diff
@@ -660,8 +672,25 @@ add_custom_target( GenerateEmbeddedEpJSONSchema
                    DEPENDS generate_embeddedable_epJSON_schema )
 
 add_subdirectory(AirflowNetwork)
-#message( $<TARGET_PROPERTY:airflownetworklib,INTERFACE_INCLUDE_DIRECTORIES> )
-#include_directories($<TARGET_PROPERTY:airflownetworklib,INTERFACE_INCLUDE_DIRECTORIES>)
+
+if(LINK_WITH_PYTHON)
+  # check the python bit version here...if we are building against Python it has to match up bitwise
+  # if we aren't, then the identified interpreter can be whatever bit it wants
+  execute_process(COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/cmake/PythonGetBitSize.py" OUTPUT_VARIABLE PYTHON_BIT_SIZE)
+  if(MSVC)
+    execute_process(COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/cmake/PythonGetBitSize.py" OUTPUT_VARIABLE PYTHON_BIT_SIZE)
```

Comment:
Why did you repeat the execute_process command, you just did that 2 lines up?

Reply:
Great catch @rraustad, thanks!  I've removed this locally but I don't want to push up just this change.  I'll push it with other changes later.